### DISCUSSION TITLE
Programmare Son-Off Mini

### DISCUSSION THREAD

**Discussion by amennone** [indomus/forum#426] [Hardware & Firmware]
Ho un programmatore ft2232  è possibile programmare son-off mini .?
Quali sono i punti di connessione ? 
Il programmatore viene riconosciuto dal software Tasmotizer ?
Grazie e buon lavoro

**Comment by Scialla** [comment]
Se parli di quello WiFi, si è possibile, ma conviene farlo OTA e non collegando l'ft2232 perché le piazzole sono piccolissime e, saldando dei fili, rischi che saltino... oppure ti fai la classica molletta per tenere premuti i puntali sulle piazzole

In ogni caso c'è la guida inDomus per metterci Tasmota, guarda sul sito

**Reply by amennone** [reply]
Salve Scialla , il problema è che non riesco a mettere il son-off in modalità di accoppiamento , penso al firmware corrotto . Appena alimento il son-off il led blu accende per un attimo per poi rispegnersi .
Tenendo premuto il pulsante di reset accende il led blu per poi rispegnersi dopo poco .
Ho bisogno di riprogrammare il dispositivo . 
Sul sito non trovo niente riguardo i punti di connessione del son-off mini con ft2232.
grazie e buone cose

**Reply by snakuzzo** [reply]
Mi sembra strano che tu non riesca ad accoppiare il device tramite l'app nativa

https://indomus.it/guide/riprogrammare-un-itead-sonoff-usando-la-modalita-diy-via-ota/

**Reply by Scialla** [reply]
![sonoff-mini-tasmota-GPIO-1024x512](https://user-images.githubusercontent.com/49528981/122038854-548aa580-cdd6-11eb-95ad-e9adc097c653.jpg)

**Reply by amennone** [reply]
Grazie per lo schema allegato . 
Sei cosi cortese da indicarmi i punti di  collegamento con il programmatore FT2232 ?
Grazie e buon lavoro

**Reply by Scialla** [reply]
foto di sx, in alto, GND-GND 3V3-3V3 TX-RX RX-TX. Non saldare perché la piazzole saltano che è una bellezza, fatti una pinza con una molletta o qualcosa del genere

**Reply by amennone** [reply]
Salve Scialla ho collegato FT2232 al son-off  ,ma non riesco a dialogare con il programma  Tasmotizer . 
Puoi postare una guida dettagliata e dei test di connessione da eseguire con altri software ? 
Grazie e buone cose

**Reply by Scialla** [reply]
Mi spiace, non ne ho, però se googli dovresti trovare qualcosa..

**Reply by essegi65** [reply]
hai provato con nodeMCU? io ho "tasmotizzato" tutti i miei sonoff così.
